### DISCUSSION TITLE
Writing tests for a whiteboard - How can I get, create, move, and manage elements on a canvas in WebGL from Cypress?

### DISCUSSION THREAD

**Discussion by cypressbeginner** [cypress-io/cypress#24536] [Questions and Help]
I need to create some tests in Cypress for a whiteboard canvas with WebGL in Chrome, starting with the basics. These would be functional tests to start, so it would mean drawing, locating, and moving these elements. No ides how to get started. Any experience in this or ideas would be much appreciated. Thank you.

Not sure how to get started with this.

**Comment by fakob** [comment]
We are trying to do the same thing for a WebGL application which uses https://pixijs.com. So far, I was not able to do this. I fail to get any mouse events in :-(
